internal/exec/packer/packer.go

```diff
@@ -325,7 +325,10 @@ func unmarshalInput(typ reflect.Type, input interface{}) (interface{}, error) {
 	if reflect.TypeOf(input) == typ {
 		return input, nil
 	}
-
+	if typ.String() == "time.Time" {
```

Comment:
may be we can make it in more flexible way?
what if I want to import package `time` like? 
```go
import . "time"
```
or 
```go
import stdtime "time"
```

Reply:
No, I wouldn't add alias to standard library packages and I would definitely avoid `.` imports. https://github.com/golang/go/wiki/CodeReviewComments#import-dot

Reply:
Also:
> Avoid renaming imports except to avoid a name collision;...

https://github.com/golang/go/wiki/CodeReviewComments#imports